core/src/main/java/org/primefaces/extensions/component/timer/Timer.java

```diff
@@ -345,6 +346,14 @@ public void setForward(final boolean forward) {
         getStateHelper().put(PropertyKeys.forward, forward);
     }
 
+    public java.lang.String getLocale() {
+        return (java.lang.String) getStateHelper().eval(PropertyKeys.locale, "en");
+    }
```

Comment:
Default value should be NULL instead of "en" and the above CalculateLocale will figure out the default.